package.json

```diff
@@ -88,5 +88,10 @@
       "./script/list-of-methods",
       "preset-wooorm"
     ]
+  },
+  "engines": {
+    "node": ">=8",
+    "npm": ">=6",
+    "yarn": ">=1"
```

Comment:
`unified` projects avoid setting the engines field, see https://github.com/unifiedjs/unified/issues/70#issuecomment-536666466 and https://github.com/unifiedjs/unified/issues/70#issuecomment-539406229 for details.

Reply:
So you want these removed then? 🤔